app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -499,15 +499,57 @@ def clone_vm(raw_args)
     end
 
     def console(uuid)
+      return vnc_console(uuid) if console_type == 'vnc'
+      webmks_console(uuid)
+    end
+
+    def vnc_console(uuid)
       vm = find_vm_by_uuid(uuid)
       raise "VM is not running!" unless vm.ready?
-      #TOOD port, password
-      #NOTE this requires the following port to be open on your ESXi FW
```

Comment:
I just don't think these comments will help anybody and wanted to clean that up. "Always leave the campground cleaner than you found it."